recline.field.inc

```diff
@@ -439,18 +437,18 @@ function recline_default_formatter_output($variables) {
     $link_text = $file->description;
     $options['attributes']['title'] = check_plain($file['filename']);
   }
-  $dkan = $table = FALSE;
+  $datastoreStatus = $table = FALSE;
```

Comment:
Adds the datastore status to define the strategy to render the preview. If the datastore api should be used or not.